tests/codegen_testcases/solidity/common_subexpression_elimination.sol

```diff
@@ -484,57 +475,56 @@ contract c1 {
     }
 
     // BEGIN-CHECK: c1::function::test16
-    function test16(int a, int b) public pure returns (int) {
-        int k = (a-b);
-         bool e = k>0;
+    function test16(int256 a, int256 b) public pure returns (int256) {
+        int256 k = (a - b);
+        bool e = k > 0;
 
-        for(int i=1; a-b < 0; i++) {
+        for (int256 i = 1; a - b < 0; i++) {
             // CHECK: ty:int256 %4.cse_temp = (signed divide %k / (arg #0))
             // CHECK: ty:int256 %p = ((%1.cse_temp * int256 5) - %4.cse_temp)
-            int p = (a-b)*5-k/a;
+            int256 p = (a - b) * 5 - k / a;
             b++;
             // CHECK: ty:int256 %1.cse_temp = ((arg #0) - %b)
             // CHECK: branchcond (signed less %1.cse_temp < int256 0), block1, block4
             // CHECK: 	ty:int256 %2.cse_temp = ((arg #0) - %b)
             // CHECK: branchcond (signed more %2.cse_temp > int256 0), block6, block7
-            while(a-b > 0) {
+            while (a - b > 0) {
                 // CHECK: ty:int256 %p = (%2.cse_temp * int256 5)
-                p = (a-b)*5;
+                p = (a - b) * 5;
                 b--;
             }
-            bool e2 = p<1;
+            bool e2 = p < 1;
         }
 
         do {
             // CHECK: ty:int256 %p.170 = ((((arg #0) - %b) * int256 5) - %4.cse_temp)
-            int p = (a-b)*5-k/a;
+            int256 p = (a - b) * 5 - k / a;
             b++;
-            bool e2 = p<1;
+            bool e2 = p < 1;
             // CHECK: branchcond (signed less ((arg #0) - %b) < int256 0), block8, block10
-        }while(a - b < 0);
+        } while (a - b < 0);
 
-        int g = b;
+        int256 g = b;
         // CHECK: ty:uint256 %p1 = (uint256((arg #0)) ** uint256(%g))
-        uint p1 = uint(a)**uint(g);
+        uint256 p1 = uint256(a)**uint256(g);
         bool e9 = p1 == 0;
         // CHECK: ty:int256 %3.cse_temp = ((arg #0) - %b)
         // CHECK: branchcond (signed less %3.cse_temp < int256 0), block12, block13
-        while(a - b < 0) {
+        while (a - b < 0) {
             // CHECK: = ((%3.cse_temp * int256 5) - %4.cse_temp)
-            int p = (a-b)*5-k/a;
-            b=4;
+            int256 p = (a - b) * 5 - k / a;
+            b = 4;
             // CHECK: ty:int256 %5.cse_temp = ((arg #0) - int256 4)
             // CHECK: branchcond (signed more %5.cse_temp > int256 0), block14, block15
-            if (a-b > 0) {
+            if (a - b > 0) {
                 // CHECK: return (%4.cse_temp + int256(%p1))
                 // CHECK:  = (%5.cse_temp * int256 4)
-                p = (a-b)*4;
+                p = (a - b) * 4;
                 b++;
             }
-            bool e2 = p<1;
+            bool e2 = p < 1;
         }
 
-        return k/a + int(uint(a)**uint(g));
+        return k / a + int256(uint256(a)**uint256(g));
```

Comment:
Why did you change `int` to `int256`?

Reply:
The solidity formatter did this. It is a lint